### PR TITLE
Add playbook for updating AzureMFA institution config only

### PR DESCRIPTION
**Pull request by phavekes**
Add a playbook to update the AzureMFA gssp config without redeploy

### CODE REVIEW

File: webauth-certificates.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  tasks:
+    - name: Find path for webauthn GSSP
+      find:
+        paths: /opt/www
+        file_type: any
+        recurse: no
+        follow: yes
+        patterns: "webauthn.*"
+      register: dirs_matched
```
Comment (pmeulen) [inline webauth-certificates.yml:14]: See #177: Why not use the info from group_vars?

**Review by pmeulen — COMMENTED** [review]
azuremfa-config.yml  is a lot of duplication. The deploy script already has the option to skip the untar option. This looks very similar to what you are doing here.

File: webauth-certificates.yml (hunk 1 above)
Comment (phavekes) [reply webauth-certificates.yml]: Changed to build the path from group_vars

**Comment by phavekes** [comment]
> azuremfa-config.yml is a lot of duplication. The deploy script already has the option to skip the untar of the component. This looks very similar to what you are doing here.

I'm looking for a way to safely update config without interruptions for the user. e.g: These playbooks wil never reload php-fpm

**Comment by phavekes** [comment]
This wil resolved by reusing the current playbook

**Pull request closed without merging**